.gitpod.Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM gitpod/workspace-full
```

Comment:
Can we move all .gitpod.* things not necessary in the root into a subfolder please?

Reply:
fixed by @gnunicorn in https://github.com/TeamFranka/affinity/commit/c6abefd546d703dd1bdc0af9e78839776e9170d3#diff-050265bb25fc16211ce409bd85cd699db1be9296267f976eb0f3cd85c5ab7ff9